src/buttons/expand-error.js

```diff
@@ -0,0 +1,43 @@
+const Discord = require('discord.js');
+const checkForErrorCode = require("../utils/errorCode").checkForErrorCode;
```

Comment:
For consistencies sake this should be written as `const { checkForErrorCode } = require('../utils/errorCode');`